docs/platforms/kotlin/index.mdx

```diff
@@ -2,8 +2,6 @@
 title: Kotlin
 caseStyle: camelCase
 supportLevel: production
-sdk: sentry.java
-fallbackPlatform: java
```

Comment:
needed to remove these, otherwise the KMP guide would use platform-includes from Java for some reason. in any case, removing this seems correct to me